host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -440,7 +443,7 @@ static void default_status(RASPIVID_STATE *state)
    state->save_pts = 0;
 
    state->netListen = false;
-
+   state->slices = 1;
```

Comment:
The structure is memset to 0, and your check later is that slices>1, therefore you don't need to initialise this to 1.

Reply:
I believe it's better to set slices to 1 by default even if it has no effect:
1) 0 as a number of slices a video stream has does not make sense. If someone uses state->slices after this, they would have to special case that 0 actually means 1 slice.
2) it's consistent with the style that is already there (why should slices be excluded from initialization?)
3) it documents that the default behavior is to encode with one, full-height slice.